src/backend/distributed/citus--5.2-4--6.0-1.sql

```diff
@@ -0,0 +1,5 @@
+/* citus--5.2-4--6.0-1.sql */
+
+/* change logicalrelid type to regclass to allow implicit casts to text */
```

Comment:
I'm slightly inclined to also add a comment to citus.sql mentioning that this is now a different type.